.rubocop.yml

```diff
@@ -1,13 +1,51 @@
 inherit_from: .rubocop_todo.yml
 
-# rspec expect{...} is conventional
-Style/SpaceBeforeBlockBraces:
+AllCops:
+  DisplayCopNames: true
+  Include:
+    - Rakefile
+    - lib/**/*.rake
+  Exclude:
+    - Gemfile.lock
```

Comment:
are `.lock` `.md` or `.yml` files ever parsed by rubocop? I've never seen this before.

Reply:
Good point - filed away for future reference.